storage/src/main/java/org/apache/kafka/storage/internals/util/CleanShutdownFileHandler.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package kafka.log;
+package org.apache.kafka.storage.internals.util;
```

Comment:
Why did we put this under `util`?

Reply:
Do you mean it should have its own package? Or any other place it put it?

Reply:
I was asking if it would fit under one of the existing pcakages: `checkpoint` or `log`.

Reply:
Done. Moved to log